releases/release-5.2.0.md

```diff
@@ -0,0 +1,329 @@
+
+    The Lock View feature provides more information about lock conflicts and lock waits of pessimistic locks, which helps DBAs to observe transaction locking events and troubleshoot deadlock problems.
+
+    In v5.2, the following enhancements are made to Lock View:
+
+    - In addition to the SQL digest column in the Lock View-related tables, add a column to these tables that shows the corresponding normalized SQL text. You do not have to manually query the statement corresponding to a SQL digest.
+    - Add the `TIDB_DECODE_SQL_DIGESTS` function to query the normalized SQL statements (a form without formats and arguments) corresponding to a set of SQL digests in the cluster. This simplifies the operation of querying the statements that have been historically executed by a transaction.
+    - Add a column in the `DATA_LOCK_WAITS` and `DEADLOCKS` system tables to show the table name, row ID, index value, and other key information interpreted from a key. This simplifies the operations such as locating the table to which a key belongs and interpreting the key information.
+    - Support collecting the information of retryable deadlock errors in the `DEADLOCKS` table, which makes it easier to troubleshoot issues caused by such errors. The error collection is disabled by default and can be enabled using the `pessimistic-txn.deadlock-history-collect-retryable` configuration.
+    - Support distinguishing query-executing transactions from idle transactions on the `TIDB_TRX` system table. The `Normal` state is now divided into `Running` and `Idle` states.
+
+   User documents:
+
+    - View the pessimistic lock-waiting events that are occurring on all TiKV nodes in the cluster: [`DATA_LOCK_WAITS`](/information-schema/information-schema-data-lock-waits.md)
+    - View the deadlock errors recently occurred on a TiDB node: [`DEADLOCKS`](/information-schema/information-schema-deadlocks.md)
+    - View the executing transaction on a TiDB node: [`TIDB_TRX`](/information-schema/information-schema-tidb-trx.md)
+
+    - Optimize the user scenarios of adding indexes on tables with the `AUTO_RANDOM` or `SHARD_ROW_ID_BITS` attribute.
+
+### Stability
+
+- **Add TiFlash I/O traffic limit**
+
+    This new feature is suitable for cloud storage with disk bandwidth of a small and specific size. It is disabled by default.
+    
+    TiFlash IO Rate Limiter provides a new mechanism to avoid excessive race for I/O resources between read and write tasks. It balances the responses to read and write tasks, and limits the rate automatically according to read/write workload.
+    
+    [User document](/tiflash/tiflash-configuration.md)
+
+- **Improve stability of TiKV flow control**
+ 
+    TiKV introduces a new flow control mechanism to replacesupersede the previous RocksDB write stall mechanism. This mechanism controls flow at the scheduler layer, which avoids the issue of QPS drop caused by the stuck Raftstore or Apply threads when the write traffic is high.
```

Comment:
```suggestion
    TiKV introduces a new flow control mechanism to replace the previous RocksDB write stall mechanism. Compared with the write stall mechanism, this new mechanism reduces the impact on the stability of foreground write by the following:

    When RocksDB compaction stacks, flow control is performed on the TiKV scheduler layer instead of the RocksDB layer, to avoid the following issues:
    
        - Raftstore is stuck, which is caused by RocksDB write stall.
        - Raft election times out, and the node leader is transferred as a result. 

    This new mechanism improves the flow control algorithm to mitigate QPS decrease when the write traffic is high.
```